UnitTests/TestSuites/ImagesTests.swift

```diff
@@ -80,5 +80,27 @@ class ImagesTests: XCTestCase {
 
     let expected = self.fixtureString("Images-File-CustomName.swift.out")
     XCTDiffStrings(result, expected)
-  }
+	}
+
+	func testFileWithDotSyntax() {
```

Comment:
Seems like you don't have the same indentation settings as the ones defined in the project (2-spaces indentation)?